src/test/fixtures/profiles/make-profile.js

```diff
@@ -112,68 +112,73 @@ export function getProfileWithNamedThreads(threadNames: string[]): Profile {
   return profile;
 }
 
-export function getEmptyThread(overrides: ?Object): Thread {
```

Comment:
using `Object` and `Object.assign` made it so that Flow was blind about it.

Reply:
Ah dang, nice catch.